akash1/HTML/ASSIn7/profession.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>profession</title>
+        <link rel="stylesheet" href="style2.css" />
+        <link href="https://fonts.googleapis.com/css2?family=Baloo+Bhaina+2&display=swap" rel="stylesheet" />
+    </head>
+    <body>
+        <header>
+            <ul>
+                <li><a class="active" href="home.html">Home</a></li>
+                <li><a href="personal.html">Personal Details</a></li>
+                <li><a href="#">Professional Details</a></li>
+                <li><a href="login.html">login</a></li>
+            </ul>
+        </header>
+        <div class="block6">
+            <div class="block4">
+                <h1>Professional Details</h1>
+            </div>
+            <hr />
+            <div class="block5">
+                <h3>Work Experience</h3>
+                <hr />
+                <label for="company"><h3>Company</h3></label>
+                <input type="text" name="company" /><br />
+                <label for="jobtitle"><h3>Job Title</h3></label>
+                <input type="text" name="jobtitle" placeholder="Job Title" /><br />
```

Comment:
remove the / at the end of line

Reply:
it is automatically come when identitatione apply